src/components/layout/Header/Header.styled.js

```diff
@@ -57,7 +57,7 @@ export const StyledUserImg = styled.div`
     position: absolute;
     top: 0;
     right: 0;
-    z-index: 999;
+    z-index: 1;
```

Comment:
What for this z-index?

Reply:
> 

That's why I added modal. Can't do without z-index)